### DISCUSSION TITLE
request.options.HostErrorsCache != nil   always  true

### DISCUSSION THREAD

**Discussion by faf-xff** [projectdiscovery/nuclei#2434] [Q&A]
The hosterrorscache in the executeroptions of the structure is the interface type.

When
var cache *hosterrorscache.Cache
request.Options.Hosterrorscache = cache
request. Options HostErrorsCache != nil ;   result is true

**Comment by olearycrew** [comment]
@guanyufen123 I'm not sure I understand the question here. Can you expand on what you're asking?

**Reply by MetzinAround** [reply]
Feel free to reopen this if you need more help!
